chat.js

```diff
@@ -12,35 +13,95 @@ const chatLog = [];
 let first_person = '';
 let second_person = '';
 let current_person = '';
+let fileChoosen = '';
 
-rl.question('Enter first person name: ', (name1) => {
-    first_person = name1;
-    rl.question('Enter second person name: ', (name2) => {
-        second_person = name2;
-        current_person = first_person;
-        chat();
-    });
+rl.question('Continue from previous chat? y/n :', (cont) => {
```

Comment:
**suggestion:** tests show that currently leaving the answer empty defaults to `n`. That's perfect, that's how it should behave. I just think we need to highlight that the way it's usually done in CLI interfaces — by uppercasing the default options. So here's what I propose:
```suggestion
rl.question('Continue from previous chat? y/N :', (cont) => {
```

Reply:
Yes sir, I just updated this for the Y and N options. It definitely makes a lot more sense.